modules/bibdocfile/lib/bibdocfile_webinterface.py

```diff
@@ -164,6 +165,13 @@ def getfile(req, form):
             else:
                 docname = args['docname']
 
+            if CFG_WEBSEARCH_OBELIX_REDIS:
+                try:
+                    from invenio.search_engine_obelix import log_download_after_search
+                    log_download_after_search(user_info, self.recid)
+                except:
```

Comment:
Any exception should be already caught already inside  `log_download_after_search`, do you really need to put it again here?